go/teams/handler.go

```diff
@@ -260,11 +260,11 @@ func handleChangeSingle(ctx context.Context, g *libkb.GlobalContext, row keybase
 
 	err = HintLatestSeqno(mctx, row.Id, row.LatestSeqno)
 	if err != nil {
-		return err
+		mctx.Debug("team.handleChangeSingle HintLatestSeqno error: %+v", err)
 	}
 	err = HintLatestHiddenSeqno(mctx, row.Id, row.LatestHiddenSeqno)
 	if err != nil {
-		return err
+		mctx.Debug("team.handleChangeSingle HintLatestHiddenSeqno error: %+v", err)
```

Comment:
I should've mentioned those methods already logs its own errors. So this here is double logging. If those methods' returned errors aren't used they can be dropped if you like.